msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -290,6 +299,90 @@ async def _mention_activity(self, turn_context: TurnContext):
 
 Now you can send an introduction message when your bot is first installed or added to a group or team.
 
+#### Tag mention
+
+Your bot can mention tags in text messages and Adaptive Cards posted into channels. When the bot @mentions the tag in a channel, the tag is highlighted and the people associated with the tag get notified. When a user hovers over the tag, a pop-up appears with the tag details.
+
+> [!NOTE]
+>
+> * Tag mentions are supported in Teams desktop and web clients.
+> * Tag mentions are supported in GCC and GCC-H tenants only.
+
+##### Mention tags in a text message
+
+In the `mention.properties` object, add the property `'type': 'tag'`. If the property `'type': 'tag'` isn't added, the bot treats the mention as a user mention.
+
+Example:
+
+```javascript
+​var mention = new ChannelAccount(tagId, "Test Tag"); 
+​mention.Properties = JObject.Parse("{'type': 'tag'}"); 
+​var mentionObj = new Mention 
+​{ 
+​    Mentioned = mention, 
+​    Text = "<at>Test Tag</at>" 
+​}; 
+
+​var replyActivity = MessageFactory.Text("Hello " + mentionObj.Text); 
+​replyActivity.Entities = new List<Microsoft.Bot.Schema.Entity> { mentionObj }; 
+​await turnContext.SendActivityAsync(replyActivity, cancellationToken); 
+```
+
+##### Mention tags in an Adaptive Card
+
+In the `mentioned` object, add the property `"type": "tag"` in the Adaptive Card schema.  If the property `"type": "tag"` isn't added, the bot treats the mention as a user mention.
+
+You can get the list of the tags available in the channel using the [List teamworkTags](/graph/api/teamworktag-list?view=graph-rest-1.0&tabs=http&preserve-view=true) API.
+
+Example:
+
+```json
+​{ 
+​    "type": "mention", 
+    ​"text": "<at>my tag</at>", 
```

Comment:
Let's rename it to `<at>Test Tag</at>` to make it consistent with the text message example.

Reply:
Updated.